files/setup-07-event-router.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+# Copyright 2019 VMware, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-2
+
+# Setup VMware Event Router
+
+set -euo pipefail
+
+echo -e "\e[92mDeploying VMware Event Router ..." > /dev/console
+kubectl --kubeconfig /root/.kube/config -n vmware create secret generic event-router-config --from-file=${EVENT_ROUTER_CONFIG}
+
+# Retrieve the VMware Event Router image
+VEBA_BOM_FILE=/root/config/veba-bom.json
+EVENT_ROUTER_IMAGE=$(jq -r < ${VEBA_BOM_FILE} '.["vmware-event-router"].containers[0].name')
+EVENT_ROUTER_VERSION=$(jq -r < ${VEBA_BOM_FILE} '.["vmware-event-router"].containers[0].version')
+
+if [[ "${EVENT_PROCESSOR_TYPE}" == "Knative" ]] && [[ "${KNATIVE_DEPLOYMENT_TYPE}" == "embedded" ]]; then
```

Comment:
nit: I think the only difference between these manifests (options) is that for Knative you use the service account and role binding. IMHO, both don't harm in the non-Knative deployment, so can we just use one deployment mode for the router and keep this script/YAML logic simple?

Reply:
👍